### DISCUSSION TITLE
native html forms?

### DISCUSSION THREAD

**Discussion by ralyodio** [oakserver/oak#372] [Q&A] [answered]
How can I get data from a native form method="POST" ?

**Comment by kitsonk** [comment, marked as answer]
Outside of the documentation in the [README documentation for the `.request.body()` method](https://github.com/oakserver/oak#request), the [`/example/echoServer.ts`](https://github.com/oakserver/oak/blob/main/examples/echoServer.ts#L39-L85) provides an example of how dealing with various types of bodies in the request can be accomplished.
